gen/icons/FlutterIcons.java

```diff
@@ -61,6 +62,9 @@ private static Icon load(String path) {
   public static final Icon Styling = load("/icons/inspector/atrule.png");
   public static final Icon Text = load("/icons/inspector/textArea.png");
 
+  public static final Icon ExpandProperty = load("/icons/inspector/expand_property.png");
```

Comment:
Expand and collapse icons used for properties so that collapsible content for properties looks different from regular tree nodes. Tao suggested these icons.